### PR TITLE
Feature/clear phone number cx 1515

### PR DESCRIPTION
**Pull request by stefaniacardenas**
* Removed phone number value from input field when clicking "back" or "Resend link"
* Fixed phone number not being displayed on SMS confirmation screen. Regression introduced by PR #258

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-270-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://270-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/crossDevice/MobileNotificationSent/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const MobileNotificationSent = (props) => {
   return (
     <div className={theme.thickWrapper}>
       <h1 className={theme.title}>Check your mobile</h1>
-      <p className={style.submessage}>We’ve sent a secure link to {props.mobileNumber}</p>
+      <p className={style.submessage}>We’ve sent a secure link to {props.sms.number}</p>
```
Comment (hjheath) [inline src/components/crossDevice/MobileNotificationSent/index.js]: my bad!

File: src/components/PhoneNumberInput/index.js [hunk 2]
```diff
@@ -32,7 +32,6 @@ class PhoneNumberInput extends Component {
   render = () =>
     <PhoneNumber placeholder='Enter mobile number'
       onChange={this.onChange}
-      value={this.props.sms.number}
```
Comment (hjheath) [inline src/components/PhoneNumberInput/index.js]: 2 story points for this!

Comment (stefaniacardenas) [reply src/components/PhoneNumberInput/index.js]: Two points cause I wasn't sure if something had changed on the redux side. And in fact, there was a small change, hence spotted the regression.

**Pull request merged**